hilla-jandex/pom.xml

```diff
@@ -57,7 +57,7 @@
                         </goals>
                         <configuration>
                             <includes>
-                                **/dev/hilla/**/*.class,
+                                **/dev/hilla/**/*.class,**/com/vaadin/**/*.class
```

Comment:
Becuse without the `vaadin-quarkus-extension` dependency, we do not have the Vaadin jandex index.
But if we add a dependency to the index, the build fails in Hilla only projects because Vaadin components classes are missing,
So we need to index the Flow classes defined as Hilla dependencies.